modules/core/src/viewports/viewport.js

```diff
@@ -366,6 +359,26 @@ export default class Viewport {
   }
   /* eslint-enable complexity, max-statements */
 
+  _getCenterInWorld({longitude, latitude}) {
```

Comment:
This still looks somewhat mercator specific (At least the meter offset parts.)

Is the intention just to collect this in a method that can be overridden? If so maybe this method should be a little more explicit, as a maintainer typically assumes he/she can modify/delete private underscore methods without considering code outside this class. At the very least a comment....

Reply:
The intention is to use `this.projectFlat` instead of `lngLatToWorld`. Custom geospatial viewports will provide their own `projectFlat` and `unprojectFlat`.

The meter offset part is only used by first-person and third-person viewports. Maybe they do not belong in the base class.